app/views/admin/events/_form.html.erb

```diff
@@ -101,6 +100,9 @@
       theme: 'snow'  // or 'bubble'
     });
 
+    var postContentInput = document.querySelector('#event_event_info_attributes_description');
```

Comment:
Where is this CSS is actually being set?

Reply:
Just asking because I don't really understand how it can work ahahaha

Reply:
I should have clarified!

During runtime, the code `<%= event_info_form.hidden_field :description, required: true %>
` gets resolved to have `id="event_event_info_attributes_description"`.

The following example shows the original view in the `development` (as of 7c6f100b0bed17c054e6ccb1a08a89079ca6851e). Which shows that `<%= event_info_form.text_area :description, required: true %>` has the `id="event_event_info_attributes_descrition"`
![](https://imgur.com/iywouu3.png)

I agree that it is quite magical with rails doing its own job, but so far this is the best option I have found. During my testing in my own sandbox (unrelated to gapura) I had this code snippet with a Post model:
```ruby
<h1>Add Post</h1>
<%= form_for :post, url: posts_path, remote: true, html: { id: 'my-form' } do |f| %>
  <p>
    <%= f.label :title %><br>
    <%= f.text_field :title %>
  </p>

  <p>
    <%= f.label :body %><br>
    <div id="editor-container" hidden="true">   
      <%= f.text_area :body %>
    </div>
    <div id="editor"></div>
  </p>

  <p>
    <%= f.submit %>
  </p>
<% end %>
```

and I found that ` var postContentInput = document.querySelector('#post_body');` referred to the line `<%= f.text_area :body %>`.

Reply:
I see, okok thanks for the clarif hahahha

Reply:
I think I should add this under comments in the code for future maintainers. Currently I’m outside; will add it tomorrow!

Reply:
Okok, no problem, I have added it :)